utils/src/ooinstall/openshift_ansible.py

```diff
@@ -175,30 +184,20 @@ def write_proxy_settings(base_inventory):
         pass
 
 
-# pylint: disable=too-many-branches
 def write_host(host, role, inventory, schedulable=None):
     global CFG
 
     if host.preconfigured:
         return
 
     facts = ''
-    if host.ip:
-        facts += ' openshift_ip={}'.format(host.ip)
-    if host.public_ip:
-        facts += ' openshift_public_ip={}'.format(host.public_ip)
-    if host.hostname:
-        facts += ' openshift_hostname={}'.format(host.hostname)
-    if host.public_hostname:
-        facts += ' openshift_public_hostname={}'.format(host.public_hostname)
-    if host.containerized:
-        facts += ' containerized={}'.format(host.containerized)
+    for prop in HOST_VARIABLES_MAP:
+        if getattr(host, prop):
+            facts += ' {}={}'.format(HOST_VARIABLES_MAP.get(prop), getattr(host, prop))
+
     if host.other_variables:
         for variable, value in host.other_variables.iteritems():
             facts += " {}={}".format(variable, value)
-    if host.node_labels:
-        if role == 'node':
-            facts += ' openshift_node_labels="{}"'.format(host.node_labels)
```

Comment:
I think that removing this specific conditional fact append (`host.node_labels`) is causing errors for me.

Running RPM from latest master I get a pretty `hosts` file like this

```ini
[nodes]
192.168.0.14  openshift_ip=192.168.0.14 openshift_public_ip=192.168.0.14 \
    openshift_hostname=deepfryer.redhat.com openshift_public_hostname=deepfryer.redhat.com \
    openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
```

See how `openshift_node_labels` sets its value wrapped in double quotes?

Then, running from [smunilla/BZ1368414](https://github.com/smunilla/openshift-ansible/commit/5fa778d12eb223bab9c9ad33c8a14cd9f1846f37)

```ini
[nodes]
192.168.0.14  openshift_node_labels={'region': 'infra'} \
    openshift_schedulable=True
```

I suspect this is what's causing me to get this (misleading) error:

```
*** Installation Summary***

...
Gathering information from hosts...
ERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/tbielawa/.config/openshift/hosts': line 2, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


[OSEv3:children]
      ^ here

/home/tbielawa/.config/openshift/hosts:13: Expected key=value host variable assignment, got: infra}
There was a problem fetching the required information. Please see /tmp/ansible.log for details.
```